packages/ui/components/avatar/Avatar.tsx

```diff
@@ -66,27 +64,7 @@ export function Avatar(props: AvatarProps) {
             )}
           </>
         </AvatarPrimitive.Fallback>
-        {props.accepted && (
-          <div
-            className={classNames(
-              "text-inverted absolute bottom-0 right-0 block rounded-full bg-green-400 ring-2 ring-white",
-              size === "lg" ? "h-5 w-5" : "h-2 w-2"
-            )}>
-            <div className="flex h-full items-center justify-center p-[2px]">
-              {size === "lg" && <Check />}
-            </div>
-          </div>
-        )}
```

Comment:
I looked through the code base for `accepted={` and only found it being used by `AvatarGroup`. That was the only result so I'm assuming that this wasn't being used.